src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -67,24 +65,41 @@ export const GenericTable = ({
   const [isConfirmRowsDeletionDialogOpen, setConfirmRowsDeletionDialogOpen] = useState(false);
   const selectedRowsRef = useRef();
 
-  const tableLabels = {
-    label: t(TranslationUtils.getParameterTranslationKey(parameterId), parameterId),
-    loading: t('genericcomponent.table.labels.loading', 'Loading...'),
-    clearErrors: t('genericcomponent.table.button.clearErrors', 'Clear'),
-    errorsPanelMainError: t('genericcomponent.table.labels.fileImportError', 'File load failed.'),
-    placeholderTitle: canChangeRowsNumber
-      ? t('genericcomponent.table.labels.addRowPlaceholderTitle', 'Import or create your first data')
-      : t('genericcomponent.table.labels.placeholderTitle', 'Import your first data'),
-    placeholderBody: canChangeRowsNumber
-      ? t(
+  const [placeholderTitle, setPlaceholderTitle] = useState();
+  const [placeholderBody, setPlaceholderBody] = useState();
```

Comment:
As far as I can see, we never set `placeholderTitle` and `placeholderBody` separately, maybe it's better to use one `useState` for both, with an object with keys `title` and `body` ?